doc/radosgw/config.rst

```diff
@@ -128,319 +130,509 @@ you have created all of the foregoing pools::
+
+Test S3 access
+--------------
+
+You need to write and run a Python test script for verifying S3 access. The S3
+access test script will connect to the ``radosgw``, create a new bucket and list
+all buckets. The values for ``aws_access_key_id`` and ``aws_secret_access_key``
+are taken from the values of ``access_key`` and ``secret_key`` returned by the
+``radosgw_admin`` command.
+
+Execute the following steps:
+
+#. You will need to install the ``python-boto`` package.
+
+   For Debian-based distros, run::
+
+		sudo apt-get install python-boto
+
+   For RPM-based distros, run::
+
+		sudo yum install python-boto
+
+#. Create the Python script::
+
+	vi s3test.py
+
+#. Add the following contents to the file::
+
+	import boto
+	import boto.s3.connection
+	access_key = 'I0PJDPCIYZ665MW88W9R'
+	secret_key = 'dxaXZ8U90SXydYzyS5ivamEP20hkLSUViiaR+ZDA'
+	conn = boto.connect_s3(
+	aws_access_key_id = access_key,
+	aws_secret_access_key = secret_key,
+	host = '{hostname}',
+	is_secure=False,
+	calling_format = boto.s3.connection.OrdinaryCallingFormat(),
+	)
+	bucket = conn.create_bucket('my-new-bucket')
+	for bucket in conn.get_all_buckets():
+		print "{name}\t{created}".format(
+			name = bucket.name,
+			created = bucket.creation_date,
+	)
+
+   Replace ``{hostname}`` with the hostname of the host where you have
+   configured the gateway service i.e, the ``gateway host``.
+
+#. Run the script::
+
+	python s3test.py
+
+   The output will be something like the following::
+
+		my-new-bucket 2015-02-16T17:09:10.000Z
+
+Test swift access
+-----------------
+
+Swift access can be verified via the ``swift`` command line client. The command
+``man swift`` will provide more information on available command line options.
+
+To install ``swift`` client, execute the following::
+
+	sudo yum install python-setuptools
+	sudo easy_install pip
+	sudo pip install --upgrade setuptools
+	sudo pip install --upgrade python-swiftclient
+
+To test swift access, execute the following::
+
+	swift -A http://{IP ADDRESS}/auth/1.0 -U testuser:swift -K ‘{swift_secret_key}’ list
+
+Replace ``{IP ADDRESS}`` with the public IP address of the gateway server and
+``{swift_secret_key}`` with its value from the output of
+``radosgw-admin key create`` command executed for the ``swift`` user.
+
+For example::
+
+	swift -A http://10.19.143.116/auth/1.0 -U testuser:swift -K ‘244+fz2gSqoHwR3lYtSbIyomyPHf3i7rgSJrF/IA’ list
+
+The output should be::
 
+	my-new-bucket
 
 
-.. _Dnsmasq: https://help.ubuntu.com/community/Dnsmasq
 .. _Configuration Reference - Pools: ../config-ref#pools
 .. _Pool Configuration: ../../rados/configuration/pool-pg-config-ref/
 .. _Pools: ../../rados/operations/pools
 .. _User Management: ../../rados/operations/user-management
-.. _Operating a Cluster: ../../rados/rados/operations/operating
+.. _backport support for unix domain sockets: https://bugs.launchpad.net/ubuntu/+source/apache2/+bug/1411030
```

Comment:
Let's change this "backport" link to specify that this is for "Ubuntu Trusty" specifically.